Set/Game/GameView.swift

```diff
@@ -34,8 +38,36 @@ struct GameView: View {
                         .foregroundColor(.white)
                         .cornerRadius(10)
                 }))
+            HStack {
+                Button {
+                    gameViewModel.didSelect(player: gameViewModel.player1)
+                    gameViewModel.startTimer()
+                } label: {
+                    Text("Player 1")
+                        .frame(width: 95, height: 30)
+                        .background(gameViewModel.player1ButtonColor)
+                        .foregroundColor(.white)
+                        .cornerRadius(10)
+                }
+                .disabled(gameViewModel.player2.id == gameViewModel.whoseTurn?.id)
+                Spacer()
+                Text("\(gameViewModel.player1.score) - \(gameViewModel.player2.score)")
+                    .font(.largeTitle)
+                Spacer()
+                Button {
+                    gameViewModel.didSelect(player: gameViewModel.player2)
+                    gameViewModel.startTimer()
+                } label: {
+                    Text("Player 2")
+                        .frame(width: 95, height: 30)
+                        .background(gameViewModel.player2ButtonColor)
+                        .foregroundColor(.white)
+                        .cornerRadius(10)
+                }
+                .disabled(gameViewModel.player1.id == gameViewModel.whoseTurn?.id)
```

Comment:
I would have field `isPlayer1Active` and calculation have in such `computed property`, please check `player1ButtonColor` property

Reply:
what should I check for `player1ButtonColor` property?